recipes/raylib/all/conanfile.py

```diff
@@ -64,6 +64,6 @@ def package_info(self):
         if self.settings.os == "Linux":
             self.cpp_info.system_libs.extend(["m", "pthread", "dl", "rt"])
         elif self.settings.os == "Windows":
-            self.cpp_info.system_libs.append("gdi32")
+            self.cpp_info.system_libs.extend(["gdi32", "winmm"])
         elif self.settings.os == "Macos":
-            self.cpp_info.frameworks.extend(["Cocoa", "IOKit", "CoreFoundation"])
+            self.cpp_info.frameworks.extend(["Cocoa", "IOKit", "CoreVideo", "GLUT", "OpenGL"])
```

Comment:
Require `opengl/system` instead of adding OpenGL here

Also why does it link against OpenGL and glut only on macOS?

Reply:
The reference to OpenGL and GLUT comes from https://github.com/raysan5/raylib/wiki/Working-on-macOS as reference (somewhere down below on building statically).

I assume that requiring `opengl/system` will take care of the linking and I see that the glfw recipe does the same thing too. I'll make the change and ask my friend to test again when I can :+1: